Formula/zile.rb

```diff
@@ -1,9 +1,10 @@
 class Zile < Formula
   desc "Text editor development kit"
   homepage "https://www.gnu.org/software/zile/"
-  url "https://ftp.gnu.org/gnu/zile/zile-2.4.15.tar.gz"
-  mirror "https://ftpmirror.gnu.org/zile/zile-2.4.15.tar.gz"
-  sha256 "39c300a34f78c37ba67793cf74685935a15568e14237a3a66fda8fcf40e3035e"
+  url "https://ftp.gnu.org/gnu/zile/zile-2.6.0.90.tar.gz"
+  mirror "https://ftpmirror.gnu.org/zile/zile-2.6.0.90.tar.gz"
```

Comment:
looks like i missed this earlier. 

For gnu stuff, the `9x` releases stand for beta release, and sometimes, odd number release stands for beta releases as well.

Reply:
Should be fine now. The author says they'll no longer publish betas to https://ftp.gnu.org/gnu/zile, so I don't think we need to worry about this anymore. That said, I still left the comment to check that the tarball is not a pre-release.